test/integration/controller/core/clusterqueue_controller_test.go

```
@@ -196,7 +196,9 @@ var _ = ginkgo.Describe("ClusterQueue controller", func() {
 				gomega.Eventually(func() error {
 					var newWL kueue.Workload
 					gomega.Expect(k8sClient.Get(ctx, client.ObjectKeyFromObject(w), &newWL)).To(gomega.Succeed())
-					newWL.Status.Admission = admissions[i]
+					if admissions[i] != nil {
+						_ = util.AdmitWorkload(&newWL, admissions[i])
```

do not silence errors. Add Expect Succeed